### DISCUSSION TITLE
Problem with Fooocus.

### DISCUSSION THREAD

**Discussion by stefanluncanu24** [lllyasviel/Fooocus#1081] [Q&A] [answered]
Hi everybody! First thing first, i'm impressed with how good and realistic Fooocus is. However i find myself in a little trouble trying to generate photos using photo input function. Every time i try to do an image input as a prompt (or even sometimes with a text promp) it gives me 4 unknown red errors on my screen. I have read online that this has to do with the CUDA allocation for the app. Can somebody give me any solutions? Thanks in advance!

**Comment by hotlml** [comment]
Turn off the VPN software; you seem to be browsing through a VPN proxy.

**Reply by stefanluncanu24** [reply]
Sorry for the late reply. I dont use a VPN. Could have been another reason?

**Comment by xhoxye** [comment]
Please upload a screenshot of the full console

**Reply by stefanluncanu24** [reply]
Already up-to-date
Update succeeded.
[System ARGV] ['entry_with_update.py', '--preset', 'realistic']
Loaded preset: /home/stefan/Desktop/Fooocus AI/Fooocus/presets/realistic.json
Failed to load config key: {"path_checkpoints": "/home/stefan/Desktop/AI/Fooocus/models/checkpoints"} is invalid or does not exist; will use {"path_checkpoints": "../models/checkpoints/"} instead.
Failed to load config key: {"path_loras": "/home/stefan/Desktop/AI/Fooocus/models/loras"} is invalid or does not exist; will use {"path_loras": "../models/loras/"} instead.
Failed to load config key: {"path_embeddings": "/home/stefan/Desktop/AI/Fooocus/models/embeddings"} is invalid or does not exist; will use {"path_embeddings": "../models/embeddings/"} instead.
Failed to load config key: {"path_vae_approx": "/home/stefan/Desktop/AI/Fooocus/models/vae_approx"} is invalid or does not exist; will use {"path_vae_approx": "../models/vae_approx/"} instead.
Failed to load config key: {"path_upscale_models": "/home/stefan/Desktop/AI/Fooocus/models/upscale_models"} is invalid or does not exist; will use {"path_upscale_models": "../models/upscale_models/"} instead.
Failed to load config key: {"path_inpaint": "/home/stefan/Desktop/AI/Fooocus/models/inpaint"} is invalid or does not exist; will use {"path_inpaint": "../models/inpaint/"} instead.
Failed to load config key: {"path_controlnet": "/home/stefan/Desktop/AI/Fooocus/models/controlnet"} is invalid or does not exist; will use {"path_controlnet": "../models/controlnet/"} instead.
Failed to load config key: {"path_clip_vision": "/home/stefan/Desktop/AI/Fooocus/models/clip_vision"} is invalid or does not exist; will use {"path_clip_vision": "../models/clip_vision/"} instead.
Failed to load config key: {"path_fooocus_expansion": "/home/stefan/Desktop/AI/Fooocus/models/prompt_expansion/fooocus_expansion"} is invalid or does not exist; will use {"path_fooocus_expansion": "../models/prompt_expansion/fooocus_expansion"} instead.
Failed to load config key: {"path_outputs": "/home/stefan/Desktop/AI/Fooocus/outputs"} is invalid or does not exist; will use {"path_outputs": "../outputs/"} instead.
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Fooocus version: 2.1.824
Running on local URL:  http://127.0.0.1:7865

Thanks for being a Gradio user! If you have questions or feedback, please join our Discord server and chat with us: https://discord.gg/feTf9x3ZSB

To create a public link, set `share=True` in `launch()`.
Total VRAM 5938 MB, total RAM 15618 MB
Set vram state to: NORMAL_VRAM
Disabling smart memory management
Device: cuda:0 NVIDIA GeForce RTX 3060 Laptop GPU : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra keys {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Base model loaded: /home/stefan/Desktop/Fooocus AI/Fooocus/models/checkpoints/realisticStockPhoto_v10.safetensors
Request to load LoRAs [['SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors', 0.25], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [/home/stefan/Desktop/Fooocus AI/Fooocus/models/checkpoints/realisticStockPhoto_v10.safetensors].
Loaded LoRA [/home/stefan/Desktop/Fooocus AI/Fooocus/models/loras/SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors] for UNet [/home/stefan/Desktop/Fooocus AI/Fooocus/models/checkpoints/realisticStockPhoto_v10.safetensors] with 788 keys at weight 0.25.
Loaded LoRA [/home/stefan/Desktop/Fooocus AI/Fooocus/models/loras/SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors] for CLIP [/home/stefan/Desktop/Fooocus AI/Fooocus/models/checkpoints/realisticStockPhoto_v10.safetensors] with 264 keys at weight 0.25.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.42 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 3.0
[Parameters] Seed = 1948600051100776744
[Fooocus] Downloading control models ...
[Fooocus] Loading control models ...
extra keys clip vision: ['vision_model.embeddings.position_ids']
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] Integrate a Dodge Challenger into the previously described landscape with a dramatically darkened sky. Place the car prominently in the foreground, ensuring it stands out against the moody backdrop. The Challenger should be well-lit, highlighting its sleek lines and muscular design, creating a striking contrast with the brooding, stormy sky in the background. The overall composition should convey a sense of power and intensity, merging the raw energy of the car with the foreboding nature of the darkened sky, cinematic, dynamic light, ambient, epic, detailed, intricate, elegant, highly decorated, extremely professional color draped, sharp focus, great fine detail, inspired, rich vivid colors, amazing deep aesthetic,, creative, positive, joyful, unique, fantastic, symmetry, perfect
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] Integrate a Dodge Challenger into the previously described landscape with a dramatically darkened sky. Place the car prominently in the foreground, ensuring it stands out against the moody backdrop. The Challenger should be well-lit, highlighting its sleek lines and muscular design, creating a striking contrast with the brooding, stormy sky in the background. The overall composition should convey a sense of power and intensity, merging the raw energy of the car with the foreboding nature of the darkened sky, cinematic, dynamic light, ambient, detailed, intricate, elegant, highly complex, artistic, sharp focus, color fabulous, dramatic, glorious, epic, professional winning, thought driven, stunning, creative, lush, bright, clear, awesome, futuristic, inspirational, peaceful
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Image processing ...
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.36 seconds
Requested to load Resampler
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.27 seconds
Requested to load To_KV
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.16 seconds
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 6.51 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
Killed

**Reply by stefanluncanu24** [reply]
so basically, whenever i try to use the photo input function it gives me red connections errors

**Reply by xhoxye** [reply]
There was an error reading the config.txt file and no relevant model was found

**Reply by stefanluncanu24** [reply]
@xhoxye sorry if is a rookie mistake, but i'm still confused ; could you give me some details about how could i fix this?

**Reply by alfred-delacosta** [reply]
Like what @xhoxye said, no models were found. Ensure that you have downloaded the default models and placed them in the following folder: 
/home/stefan/Desktop/AI/Fooocus/models/checkpoints

Follow the steps in the Install: https://github.com/lllyasviel/Fooocus#download and if you are not on windows, you will need to download each default model from here: https://github.com/lllyasviel/Fooocus#models and place them in that checkpoints folder.

**Reply by stefanluncanu24** [reply]
@alfred-delacosta i have seen the checkpoints folder and i found the realisticStockPhotoV10 here and a "put_checkpoints_here" txt file that is empty and might be the problem? every time i run the code with the python entry_with_update.py --preset realistic ; why they are no models found ?

**Reply by alfred-delacosta** [reply]
Interesting. It could be looking for the other presets as well. Try downloading the **juggernautXL v6_RunDiffusion** model, placing it in the checkpoints folder, and trying that. 

Also, what operating system are you using?

**Reply by jakubkz** [reply]
I have the same problem, i am using windows 10 I will try doing what you suggest and I will let u know

**Reply by lllyasviel** [reply, marked as answer]
if it says killed after model load, it means your system swap is full.
see also readme how to use Microsoft virtual swap

**Reply by jakubkz** [reply]
@stefanluncanu24 by the way, did you manage to fix this problem?

**Reply by stefanluncanu24** [reply]
Sorry for the late response. So i tried the juggernautXL and it didn't work. I am quite confused. I am using Ubuntu, how should i proceed to fix the "system swap is full" that you are talking about @lllyasviel .

**Reply by lllyasviel** [reply]
@stefanluncanu24 see the guide here https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md

**Reply by stefanluncanu24** [reply]
> if it says killed after model load, it means your system swap is full.
> 
> see also readme how to use Microsoft virtual swap

this solved my problem
thanks a lot✌️

**Comment by javidnaghiyev** [comment]
I have the same problem. I only get the errors when using realistic mode

**Comment by lllyasviel** [comment]
see also common problem troubleshoot https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md

**Comment by usman19192023** [comment]
please tell me the negative prompts because it is removed in the newer version. negative prompts in order to create clear images
